src/Compilers/CSharp/Test/Semantic/SourceGeneration/SyntaxAwareGeneratorTests.cs

```diff
@@ -841,6 +841,35 @@ class C
             Assert.Null(syntaxRx);
         }
 
+        [Fact]
```

Comment:
This was just for testing. Ignore for now and i'll update with a proper suite of tests